### PR TITLE
Update connection names from sMRIPrep's ``anat_preproc_wf``

### LINKED ISSUE #1825: Update connection names from sMRIPrep's ``anat_preproc_wf``
A recent PR in `smriprep` breaks connections between the anat and func preproc wfs:
https://github.com/poldracklab/smriprep/pull/108

Here:
https://github.com/poldracklab/fmriprep/blob/2c0a8b39af4fac7416ce5d242c45f86e4a195b91/fmriprep/workflows/base.py#L624

Should be an easy fix, I can submit a PR.

### PR DESCRIPTION
**Pull request by adelavega**
Fixes #1825

I also bumped smriprep to the current master commit.

### CODE REVIEW

**Comment by adelavega** [comment]
Looks like the connections are set. Now I'm getting this error, and I'm guessing it has to do another change that happened since the last release:

```
b41-9c38335d0e7b/crash-20191015-231556-delavega-select_std.aI.a0-6aa9a112-b6e0-4c04-9ddf-add0b742f28a.txt
Traceback (most recent call last):
  File "/usr/local/miniconda/lib/python3.7/site-packages/nipype/pipeline/plugins/multiproc.py", line 316, in _send_procs_to_workers
    self.procs[jobid].run(updatehash=updatehash)
  File "/usr/local/miniconda/lib/python3.7/site-packages/nipype/pipeline/engine/nodes.py", line 450, in run
    self._get_hashval()
  File "/usr/local/miniconda/lib/python3.7/site-packages/nipype/pipeline/engine/nodes.py", line 499, in _get_hashval
    self._get_inputs()
  File "/usr/local/miniconda/lib/python3.7/site-packages/nipype/pipeline/engine/nodes.py", line 546, in _get_inputs
    self.set_input(key, deepcopy(output_value))
  File "/usr/local/miniconda/lib/python3.7/site-packages/nipype/pipeline/engine/nodes.py", line 279, in set_input
    setattr(self.inputs, parameter, deepcopy(val))
  File "/usr/local/miniconda/lib/python3.7/site-packages/nipype/interfaces/base/traits_extension.py", line 400, in validate
    value = super(MultiObject, self).validate(objekt, name, newvalue)
  File "/usr/local/miniconda/lib/python3.7/site-packages/traits/trait_types.py", line 2336, in validate
    return TraitListObject( self, object, name, value )
  File "/usr/local/miniconda/lib/python3.7/site-packages/traits/trait_handlers.py", line 2313, in __init__
    raise excp
  File "/usr/local/miniconda/lib/python3.7/site-packages/traits/trait_handlers.py", line 2305, in __init__
    value = [ validate( object, name, val ) for val in value ]
  File "/usr/local/miniconda/lib/python3.7/site-packages/traits/trait_handlers.py", line 2305, in <listcomp>
    value = [ validate( object, name, val ) for val in value ]
  File "/usr/local/miniconda/lib/python3.7/site-packages/traits/trait_types.py", line 411, in validate
    self.error( object, name, value )
  File "/usr/local/miniconda/lib/python3.7/site-packages/traits/trait_handlers.py", line 172, in error
    value )
traits.trait_errors.TraitError: Each element of the 'keys' trait of a KeySelectInputSpec instance must be a unicode string, but a value of ('MNI152NLin2009cAsym', {}) <class 'tuple'> was specified.

Error setting node input:
Node: select_std
input: keys
results_file: /work/04578/delavega/lonestar/fmriprep/work/test/fmriprep_wf/single_subject_19_wf/anat_preproc_wf/anat_norm_wf/outputnode/result_outputnode.pklz
value: [('MNI152NLin2009cAsym', {})]
```

**Comment by oesteban** [comment]
That's a node to select across iterables (in particular, it allows spatial normalization to a number of templates). The release version already has this feature.

I'll try to go through your PR tomorrow, and help you work around this problem.

**Comment by adelavega** [comment]
Great, thanks! No rush, I've got my dataset processed. Looks like my last commit didn't work

**Comment by adelavega** [comment]
Aside from an error downloading a template from templateflows, all tests are passing.
Ready for review.

File: setup.cfg [hunk 1]
```diff
@@ -31,7 +31,7 @@ install_requires =
     pybids ~= 0.9.4
     pyyaml
     sdcflows ~= 0.1.3
-    smriprep ~= 0.3.2
+    smriprep @ git+https://github.com/poldracklab/smriprep.git@24974cb5e12d5a26aaa243bff7c26fca4437d342
```
Comment (oesteban) [inline setup.cfg]:
```suggestion
    smriprep ~= 0.4.0rc3
```

**Comment by oesteban** [comment]
Okay, we'll need to hold the horses for a little. I'm also getting https://github.com/poldracklab/fmriprep/pull/1827#issuecomment-542502578. Will try to figure out tomorrow.

File: setup.cfg [hunk 2]
```diff
@@ -31,7 +31,7 @@ install_requires =
     pybids ~= 0.9.4
     pyyaml
     sdcflows ~= 0.1.3
-    smriprep ~= 0.3.2
+    smriprep ~= 0.4.0rc3
```
Comment (oesteban) [inline setup.cfg]:
```suggestion
    smriprep ~= 0.4.0rc4
```

**Comment by oesteban** [comment]
Sorry, this got out of hand, I've picked your commits and started fresh on a different PR. Will let you know when this is ready. Thanks for your patience.

**Comment by adelavega** [comment]
No worries

**Pull request closed without merging**